src/start_env.rs

```diff
@@ -377,7 +377,17 @@ fn start_cnds(envfile_path: &PathBuf) -> impl Future<Item = Vec<Node<Cnd>>, Erro
 }
 
 fn temp_folder() -> PathBuf {
-    tempfile::tempdir_in("/tmp").unwrap().into_path()
+    let path = "/tmp/create-comit-app";
+
+    let res = std::fs::DirBuilder::new().create(&path);
```

Comment:
Use https://doc.rust-lang.org/std/fs/fn.create_dir_all.html to avoid the matching logic